src/tokens/functional/color/light/patterns-display-light.json5

```diff
@@ -0,0 +1,1035 @@
+{
+  display-blue: {
```

Comment:
Why do you do `display-blue` instead of nesting:

```js
display: {
  blue: ...,
  purple: ...,
}
```

Reply:
I tested Copilot Workspaces for this PR and that's what it did 😆 don't worry, I've fixed it